astropy/coordinates/representation.py

```diff
@@ -223,6 +223,24 @@ def _unitstr(self):
                            for component in self.components]))
         return unitstr
 
+    def __eq__(self, other):
```

Comment:
Sorry - I'm travelling on holiday so this is a bit rushed with my last bit
of internet.

I vaguely recall that there is an issue with creating GCRS frames with
obsgeoloc and obsgeovel set, and particularly set to >ND. This may have
gone away with adoption of cartesianrepresentations? If so then I'm happy
to remove _eq_

Stuart Littlefair
Dept. of Physics & Astronomy
Univ. of Sheffield, Sheffield, S3 7RH

email: s.littlefair@shef.ac.uk
Phone: +44 114 2224525
Sent from my iPhone

On 30 Aug 2016, at 21:15, Marten van Kerkwijk <notifications@github.com>
wrote:

In astropy/coordinates/representation.py
<https://github.com/astropy/astropy/pull/5207#discussion_r76870586>:

> @@ -223,6 +223,24 @@ def _unitstr(self):
>                             for component in self.components]))
>          return unitstr
>
> +    def __eq__(self, other):

@StuartLittlefair <https://github.com/StuartLittlefair> - in @eteq
<https://github.com/eteq>'s review of this in #5253
<https://github.com/astropy/astropy/pull/5253>, the addition of __eq__ came
up again, as it breaks backward compatibility. I just checked and
everything runs fine if I simply remove this again, which leaves me even
more confused. My sense right now would be to leave it out. If we do want
equality checks, we can do that in a separate PR.

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/astropy/astropy/pull/5207/files/92f42af1864227c122b61286585e5ffcedfb36f1#r76870586>,
or mute the thread
<https://github.com/notifications/unsubscribe-auth/AEW-t3nuHKKKizBYRrIURHX3mSp1UbQAks5qlI90gaJpZM4JXYsL>
.